### PR TITLE
feat:  add all CI to build theme artifact 

### LINKED ISSUE #13: [Filler Task]: Add CI to build theme artifact for all blogs
- IOTA
- Shimmer
- Assembly

command your be `yarn production`

### PR DESCRIPTION
**Pull request by evavirseda**
Closes #13

check [here](https://github.com/evavirseda/liebling/actions)

### CODE REVIEW

File: .github/workflows/deploy-theme.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  push:
+    branches:
+      - feat/iota-blog
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Deploy Ghost Theme
+        uses: TryGhost/action-deploy-theme@v1.6.1
```
Comment (begonaalvarezd) [inline .github/workflows/deploy-theme.yaml]: I think i prefer not using this action for now, this is to upload directly the theme to a server, but I think we are safer just creating the artifact and be able to download it from the gh action panel. in localhost I use `yarn production`, can we do something similar to what localhost does just to generate the zip?

**Review by begonaalvarezd — CHANGES_REQUESTED** [review]

**Review by cpl121 — APPROVED** [review]
Amazing 💯

File: .github/workflows/build-artifact-assembly.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      - name: Use Node.js 14
+        uses: actions/setup-node@v2
+        with:
+            node-version: '14'
+      - name: Install dependencies
+        run: cd src && yarn
+      - name: add webpack
+        run: npm link webpack
+      - name: Build and create zip
+        run: cd src && cd .. && zip -r liebling.zip ./* -x 'node_modules/*' -x '*src*'
```
Comment (cpl121) [inline .github/workflows/build-artifact-assembly.yaml]: Is it necessary to go into the src folder and exit? `cd src && cd ..`

File: .github/workflows/build-artifact-iota.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      - name: Use Node.js 14
+        uses: actions/setup-node@v2
+        with:
+            node-version: '14'
+      - name: Install dependencies
+        run: cd src && yarn
+      - name: add webpack
+        run: npm link webpack
+      - name: Build and create zip
+        run: cd src && cd .. && zip -r liebling.zip ./* -x 'node_modules/*' -x '*src*'
```
Comment (cpl121) [inline .github/workflows/build-artifact-iota.yaml]: same

File: .github/workflows/build-artifact-shimmer.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+      - name: Use Node.js 14
+        uses: actions/setup-node@v2
+        with:
+            node-version: '14'
+      - name: Install dependencies
+        run: cd src && yarn
+      - name: add webpack
+        run: npm link webpack
+      - name: Build and create zip
+        run: cd src && cd .. && zip -r liebling.zip ./* -x 'node_modules/*' -x '*src*'
```
Comment (cpl121) [inline .github/workflows/build-artifact-shimmer.yaml]: same

**Review by cpl121 — COMMENTED** [review]
1 question

File: .github/workflows/build-artifact-assembly.yaml (hunk 2 above)
Comment (evavirseda) [reply .github/workflows/build-artifact-assembly.yaml]: Oh, you're right. I used to have to go to `src` file when using `yarn production`. I'll change it! Thanks!

**Review by cpl121 — APPROVED** [review]
LGTM 💯

**Pull request merged**